website/content/blog/reducing-your-environmental-impact-with-the-linkerd-service-mesh.md

```diff
@@ -0,0 +1,46 @@
+#  Reducing your environmental impact with the Linkerd service mesh
+
```

Comment:
```suggestion
*Guest post as part of the [cloud native sustainability week](https://tag-env-sustainability.cncf.io/cloud-native-sustainability-week/)*

```
On the TAG ENV blog we need to highlight if a blog is from the TAG directly (an artifact from a WG for example).